services/data_plane_manager/src/main/java/com/futurewei/alcor/dataplane/client/grpc/DataPlaneClientImplV2.java

```diff
@@ -173,7 +183,8 @@ private GrpcChannelStub getOrCreateGrpcChannel(String hostIp) {
         ArrayList<GrpcChannelStub> arr = new ArrayList<>();
         for (int i = 0; i < numberOfGrpcChannelPerHost; i++) {
             GrpcChannelStub channelStub = createGrpcChannelStub(hostIp);
-            warmUpChannelStub(channelStub, hostIp);
+            // Using Linkerd load balance
+            //warmUpChannelStub(channelStub, hostIp);
```

Comment:
@zzxgzgz If we comment out the channel warmup, do we expect the gRPC performance gets impacted?

Reply:
It does, but the difference shouldn't be too big. The reason we added warmup to the channels is for better performance.

Reply:
This is for Linkerd grpc load balance. Will investigate how to do linkerd grpc warm up.

Reply:
Don't know if it still work for linkerd grpc. Will investigate it.

Reply:
Thanks. Let us use this issue for tracking https://github.com/futurewei-cloud/alcor/issues/705